samples/batch/csharp/program.cs

```diff
@@ -66,16 +66,18 @@ static async Task TranscribeAsync()
                 await client.DeleteTranscriptionAsync(item.Id).ConfigureAwait(false);
             }
 
-            Console.WriteLine("Creating transcriptions.");
             var transcriptionLocation = await client.PostTranscriptionAsync(Name, Description, Locale, new Uri(RecordingsBlobUri), modelList).ConfigureAwait(false);
+            var thisTranscriptionGuid = new Guid(transcriptionLocation.ToString().Split('/').LastOrDefault());
 
             // get the transcription Id from the location URI
             var createdTranscriptions = new List<Guid>();
-            createdTranscriptions.Add(new Guid(transcriptionLocation.ToString().Split('/').LastOrDefault()));
+            createdTranscriptions.Add(thisTranscriptionGuid);
+
+            Console.WriteLine($"Created transcription with id {thisTranscriptionGuid}.");
 
             Console.WriteLine("Checking status.");
 
-            // check for the status of our transcriptions every 30 sec. (can also be 1, 2, 5 min depending on usage)
+            // check for the status of our transcriptions every 5 sec. (can also be 1, 2, 5 min depending on usage)
```

Comment:
Can you say here "// Check for the status of our transcriptions periodically" and put the timing comments where the actual wait is? "// Check very 5 seconds (adjust according to your usage)" or smth

Reply:
much better, thanks!